hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -139,10 +139,9 @@ class PrisonerSynchroniserService(
       restrictedPatientData = Result.success(restrictedPatient),
     )
     if (prisonerDifferenceService.hasChanged(existingPrisoner, prisoner)) {
-      prisonerDifferenceService.reportDiffTelemetry(existingPrisoner, prisoner)
+      prisonerDifferenceService.reportDiffTelemetry(existingPrisoner, prisoner, label)
 
       indices.map { prisonerRepository.save(prisoner, it) }
-      prisonerRepository.save(prisoner, SyncIndex.RED)
       prisonerDifferenceService.handleDifferences(existingPrisoner, ob, prisoner, "REFRESH")
```

Comment:
this doesn't look right - you'll potentially report differences in the red index and generate events for them?

Reply:
Added if statement - not that elegant but hopefully will disappear when the new index takes over